app/controllers/dradis/plugins/export/base_controller.rb

```diff
@@ -35,6 +38,15 @@ def engine_name
         def templates_dir
           @templates_dir ||= File.join(::Configuration::paths_templates_reports, engine_name)
         end
+
+        def track_export
+          track_usage('report.exported', {
+            exporter: engine_name,
+            issue_count: current_project.issues.size,
+            evidence_count: current_project.evidence.size,
+            node_count: current_project.nodes.in_tree.size
```

Comment:
Does `#size` trigger a SQL COUNT the way `#count` does? It may be worth the change.